docs/Securing-your-IoT-from-hacking.md

```diff
@@ -92,7 +92,42 @@ pattern write tele/%u/#
 
 My user root is allowed to do everything. This is used in my home-automation to control all devices and listen to all devices. The "pattern" is used for ALL other users and the %u is a substitute. The great thing is that the device can read its configuration but cannot write to it. And the status information it posts to the /status/ but is not able to read it afterward. With this minimal configuration, Tasmota devices are running.
 
-To add the different user to Mosquitte the following two commands work fine. There is also a re-read available, but a restart works better for me.
+____
+
+**Note: If you are using Home Assistant**, and using the Tasmota integration, and if you have used `SetOption19 0` on the Tasmota device, then the devices will have to write to the `tasmota/discovery/#` topic. So, add the following to the ACL file (user section or general section): `topic write tasmota/discovery/#`.
```

Comment:
Hmm, I am a little confused. When I was using just the MQTT integration (and `SetOption19 1`), the device was not writing to the `tasmota/discovery` topic. 

But after I switched to the Tasmota integration in my Home Assistant and changed my device to `SetOption19 0`, the device wrote to the `tasmota/discovery` topic. The Tasmota integration was not able to detect these devices till I added the ACL rules to allow the devices to write to that topic.

I even raised a support request [here](https://github.com/arendst/Tasmota/discussions/12817) about this where @barbudor helped me figure this out.

Am I misunderstanding your message, @blakadder ?

Reply:
Yes you are. ACL settings are only enforced when using MQTT add-on with Home Assistant OS, not when using Home Assistant with another broker

Reply:
So, do you want me to reword this line to say the following?

==============

Note: If you are using Home Assistant OS with the MQTT add-on, and if you have used `SetOption19 0` on the Tasmota device, then the device will have to write to the `tasmota/discovery/#` topic. So, add the following to the ACL file (user section or general section): `topic write tasmota/discovery/#`.

This is necessary because the Tasmota integration in the Home Assistant OS uses the above topic to detect the devices.

===============